tools/@aws-cdk/cli-args-gen/lib/yargs-gen.ts

```diff
@@ -45,19 +47,14 @@ export async function renderYargs(config: CliConfig, helpers: CliHelpers): Promi
   parseCommandLineArguments.addBody(makeYargs(config, helpers));
 
   const ts = new TypeScriptRenderer({
-    disabledEsLintRules: [
-      '@stylistic/comma-dangle',
-      '@stylistic/comma-spacing',
-      '@stylistic/max-len',
-      '@stylistic/quotes',
-      '@stylistic/quote-props',
-    ] as any, // Force our string[] into EsLintRules[], it will work out at runtime
+    disabledEsLintRules: [EsLintRules.MAX_LEN], // the default disabled rules result in 'Definition for rule 'prettier/prettier' was not found'
   }).render(scope);
 
   return prettier.format(ts, {
     parser: 'typescript',
     printWidth: 150,
     singleQuote: true,
+    trailingComma: 'all',
```

Comment:
same comment as above that I'd prefer these be in seperate prs so we're not crossing the streams but it's fine for now